DependencyUpdated/Program.cs

```diff
@@ -50,7 +50,8 @@ private static void RunApplication(Options options)
                     repositoryProvider.SwitchToDefaultBranch(repositoryPath);
                     var projectName = configEntry.Name;
                     repositoryProvider.SwitchToUpdateBranch(repositoryPath, projectName);
-                    var updates = updater.UpdateProject(project).Result;
+                    var pathWhereToSearchFiles = Path.Combine(repositoryPath, project);
```

Comment:
What if I pass "C://Sourcers" into project path? I believe it will break?
Maybe first check if path is absolute or relative?

Reply:
I change approach. Insted of messing with paths. Im setting repository path as a working directory - so all relative paths will work and if user pass a absolute path if will be also working